[OKTA-643408] feat: add new template identify_inactive_third_party_app_users

runzeng-okta opened the pull request:
Change:
Add a new template: identify_inactive_third_party_app_users

Ticket: https://oktainc.atlassian.net/browse/OKTA-643408

Reviewer: @JirongYang-okta

JirongYang-okta reviewed (approved):
Merging new template Identify inactive third-party app users, reviewed that all screenshots, readme file, flopack, and workflow.json files are correct.
Approved.

runzeng-okta commented:
@JirongYang-okta Thanks for reviewing Commit 1 - 3. I added Commit 4 by mistake after you reviewed, then I reverted the change at commit 5. Simply to say, nothing is changed after you reviewed Commit 3, can you please approve again so that merging won't be blocked?

JirongYang-okta reviewed (approved):
New template was accidentally committed in this PR but reverted, approve again.
